design/ui/pagination/cosmos.decorator.js

```diff
@@ -0,0 +1,11 @@
+// The only thing this decorator will do for now is import our SUI custom CSS.
+// There are currently issues with the out of the box compiled styles there
+// and possibly some missing loaders so we will use a different variant for now
+// that does not have any missing font/image dependencies.
+
+// import 'semantic-ui-css-custom';
```

Comment:
Uncomment this and see that we are getting the new CSS in for all cosmos fixtures